pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -601,10 +601,10 @@ func (cConfig *CruiseControlConfig) GetCCLog4jConfig() string {
 		return cConfig.Log4jConfig
 	}
 	return `log4j.rootLogger = INFO, FILE
-    log4j.appender.FILE=org.apache.log4j.FileAppender
-    log4j.appender.FILE.File=/dev/stdout
-    log4j.appender.FILE.layout=org.apache.log4j.PatternLayout
-    log4j.appender.FILE.layout.conversionPattern=%-6r [%15.15t] %-5p %30.30c %x - %m%n`
+log4j.appender.FILE=org.apache.log4j.FileAppender
+log4j.appender.FILE.File=/dev/stdout
+log4j.appender.FILE.layout=org.apache.log4j.PatternLayout
+log4j.appender.FILE.layout.conversionPattern=[%d] %p %m (%c)%n`
```

Comment:
Could you please add a sample line with this pattern (before/after)?

Reply:
I was fine changing the pattern to `[%d] %p %m (%c)%n`, I was just curious how the new pattern actually looks like.

Based on the [log4j docs](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PatternLayout.html) logs are displayed like this:
```
[1999-11-27 15:49:37,459] INFO Log entry (org.apache.log4j.examples.Sort)
``` 

While the currently used pattern is:
```
256   [main          ] INFO  org.apache.log4j.examples.Sort      - Log entry
```

I think the one you originally wanted to use is more human readable, so I'd suggest to use that one, instead of keeping the existing one.

Reply:
I'll do that in a different PR

Reply:
Okay, thanks!